book/02-git-basics/sections/recording-changes.asc

```diff
@@ -28,7 +28,7 @@ Your branch is up-to-date with 'origin/master'.
 nothing to commit, working directory clean
 ----
 
-This means you have a clean working directory – in other words, there are no tracked and modified files.
+This means you have a clean working directory – in other words, there are no modified files among tracked ones.
```

Comment:
✨ I love this:

> This means you have a clean working directory – in other words, none of your tracked files are modified.

@DiamondeX, can you push it to a new commit on this branch?